pkgs/top-level/release-lib.nix

```diff
@@ -46,6 +46,11 @@ rec {
   pkgs_x86_64_cygwin = allPackages { system = "x86_64-cygwin"; };
 
 
+  assertTrue = bool:
+    if bool
+    then pkgs.runCommand "evaluated-to-true" {} "true"
+    then pkgs.runCommand "evaluated-to-false" {} "false";
```

Comment:
Well, I want each commit to stand on its own so that should be correct from the start!